WcaOnRails/app/views/wca_monthly_digest_mailer/send_weat_digest_content.html.erb

```diff
@@ -0,0 +1,44 @@
+<p>Dear WCA Executive Assistants Team,</p>
+
+<p>Here's the draft for this month digest (Please verify whether all the changes were informed to the WCA Board):</p>
+
+<% @teams = Team.unscoped.all %>
+<% @duration_start_date = Time.now.beginning_of_month - 12.month %>
+
+<div><b>Changes in Teams/Committees/Councils</b></div>
+
+<% @teams.each do |team| %>
```

Comment:
I am not too wild about doing data computations (determining which team members have been added/removed) within the template itself. It distracts from the actual template text too much imo.

If possible, move the data computations into the handler/controller and pass them to this mail template as variables. See our monthly WST chore email as an example for how to pass variables to an erb template.

Reply:
If you want to go the really, really clean software architecture approach, extend the `team.rb` model by methods similar to `members_added_after(date)` and `members_removed_before(date)` instead of computing these in-place. That's really only the optional icing on the cake, though :D

Reply:
I've added the logic in team.rb.